website/docs/r/log_analytics_datasource_linux_syslog_collection.html.markdown

```diff
@@ -0,0 +1,74 @@
+---
+subcategory: "Log Analytics"
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_log_analytics_datasource_linux_syslog_collection"
+description: |-
+  Manages a Log Analytics Linux Syslog Collection DataSource.
+---
+
+# azurerm_log_analytics_datasource_linux_syslog_collection
+
+Manages a Log Analytics Linux Syslog Collection DataSource.
+
+## Example Usage
+
+```hcl
+provider "azurerm" {
+  features {}
+}
+
+resource "azurerm_resource_group" "example" {
+  name     = "example-resources"
+  location = "West Europe"
+}
+
+resource "azurerm_log_analytics_workspace" "example" {
+  name                = "example-law"
+  location            = azurerm_resource_group.example.location
+  resource_group_name = azurerm_resource_group.example.name
+  sku                 = "PerGB2018"
+}
+
+resource "azurerm_log_analytics_datasource_linux_syslog_collection" "example" {
+  name                = "example-lad-wpc"
+  resource_group_name = azurerm_resource_group.example.name
+  workspace_name      = azurerm_log_analytics_workspace.example.name
+  state               = "Enabled"
```

Comment:
```suggestion
  enabled             = true
```

Reply:
Resolved to reflect new schema properties